### ISSUE TITLE
Trouble with an alternate MathJax URL and a custom template

### ISSUE THREAD

**Issue by RLesur** [rstudio/rmarkdown#1593]
I think the problem comes from **rmarkdown**.
I found a little bug in **rmarkdown** when a HTML document is rendered with a custom template and `mathjax: <some_url>`.
For now, here is a workaround for this bug. In https://github.com/topepo/MathJax-test/blob/master/_output.yml, you can use these arguments:
```yaml
bookdown::gitbook:
  css: FES.css
  dev: svglite
  split_by: section
  mathjax: null
  pandoc_args: --mathjax=https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_HTMLorMML.js
```
I will open a PR in **rmarkdown** to fix this bug.

_Originally posted by @RLesur in https://github.com/rstudio/bookdown/issues/683#issuecomment-506997150_

**Comment by RLesur** [comment]
Here is a minimal example to document the problem faced with `self_contained = FALSE` and a custom MathJax URL.
Take this `Rmd` file:

    ---
    title: "Trouble with MathJax"
    output: 
      html_document:
        template: null
        mathjax: "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_HTMLorMML.js"
        self_contained: false    
    ---
    
    $$\bar{X}=\frac{1}{n}\sum_{i=1}^nX_i$$

When rendered, the source code of the MathJax library is included in the document.
The problem comes from the Pandoc arguments generated by **rmarkdown**.
In short, the Pandoc arguments are `--mathjax URL`. 
With these arguments, Pandoc interprets as follows:
- use MathJax to render maths
- source the URL as a markdown document to be included in the final document

The correct argument should be `--mathjax=URL`.
